.github/workflows/publish.yml

```diff
@@ -103,8 +103,6 @@ jobs:
       - name: Publish package
         uses: pypa/gh-action-pypi-publish@release/v1.8
         with:
```

Comment:
We might need to grant write permissions for `token-id` in the workflow file to make sure the OCID token could be fetched.

https://github.com/pypa/gh-action-pypi-publish?tab=readme-ov-file#trusted-publishing

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token